types/requests.ts

```diff
@@ -35,12 +35,21 @@ export interface BaseParams {
    * Presence penalty applied to the next token's logprobs if the token has
    * already been seen in the response.
    */
-  presencePenalty?: number
+  presencePenalty?: number;
   /**
    * Frequency penalty applied to the next token's logprobs, multiplied by the
    * number of times each token has been seen in the respponse so far.
    */
-  frequencyPenalty?: number
+  frequencyPenalty?: number;
+  /**
+   * If True, export the logprobs results in response.
+   */
+  responseLogProbs?: boolean;
```

Comment:
I'm sorry, I just looked at the proto and saw that these go inside GenerationConfig? That's down in line 104. It seems the same is the case for presencePenalty and frequencyPenalty as well.